rdbms/src/main/java/org/spine3/server/storage/jdbc/entity/query/SelectBulkQuery.java

```diff
@@ -22,91 +22,145 @@
 
 import com.google.protobuf.Descriptors;
 import com.google.protobuf.FieldMask;
-import com.google.protobuf.Message;
-import org.slf4j.Logger;
 import org.spine3.protobuf.TypeUrl;
-import org.spine3.server.storage.jdbc.query.Query;
-import org.spine3.server.storage.jdbc.util.DataSourceWrapper;
-import org.spine3.server.storage.jdbc.util.SqlExecutionHelper;
+import org.spine3.server.storage.EntityStorageRecord;
+import org.spine3.server.storage.jdbc.query.StorageQuery;
+import org.spine3.server.storage.jdbc.util.ConnectionWrapper;
 
+import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.SQLException;
+import java.util.ArrayList;
 import java.util.Iterator;
+import java.util.List;
 import java.util.Map;
 
 import static com.google.common.base.Preconditions.checkNotNull;
 
 /**
+ * <p>Implementation of {@link StorageQuery} for selecting all records or a bunch of records from a database.
```

Comment:
No `<p>` at the beginning.

Also, the first line should state what this class is generally about.